eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -23,24 +30,32 @@ foreach ($branch in $branches)
     $head = "${RepoOwner}/${RepoName}:${branchName}"
     LogDebug "Operating on branch [ $branchName ]"
     $pullRequests = Get-GitHubPullRequests -RepoOwner $RepoOwner -RepoName $RepoName -State "all" -Head $head -AuthToken $AuthToken
+    $openPullRequests = $pullRequests | ? { $_.State -eq "open" }
+
+    if ($RepoOwnerTwo)
+    {
+      $pRsAgainstRepoTwo = Get-GitHubPullRequests -RepoOwner $RepoOwnerTwo -RepoName $RepoName -State "all" -Head $head -AuthToken $AuthToken
+      $openPrsAgainstRepoTwo = $pRsAgainstRepoTwo | ? { $_.State -eq "open" }
+    }
   }
   catch
   {
     LogError "Get-GitHubPullRequests failed with exception:`n$_"
     exit 1
   }
 
-  $openPullRequests = $pullRequests | ? { $_.State -eq "open" }
-
-  if ($openPullRequests.Count -eq 0)
+  if (($openPrsAgainstRepoTwo -and $openPrsAgainstRepoTwo.Count -gt 0) -or $openPullRequests.Count -gt 0)
   {
-    LogDebug "Branch [ $branchName ] in repo [ $RepoName ] has no associated open Pull Request. Deleting Branch"
-    try{
-      Remove-GitHubSourceReferences -RepoOwner $RepoOwner -RepoName $RepoName -Ref ($branch.Remove(0,5)) -AuthToken $AuthToken
-    }
-    catch {
-      LogError "Remove-GitHubSourceReferences failed with exception:`n$_"
-      exit 1
-    }
+    LogDebug "Branch [ $branchName ] in repo [ $RepoName ] has open pull Requests. Skipping"
```

Comment:
It would be potentially help to provide a link to the open PR as well so we can easily find it if we need to go looking for it.